src/ert/gui/simulation/experiment_panel.py

```diff
@@ -81,11 +76,11 @@ def __init__(
 
         self.run_button = QToolButton()
         self.run_button.setObjectName("run_experiment")
-        self.run_button.setText(EXPERIMENT_READY_TO_RUN_BUTTON_MESSAGE)
         self.run_button.setIcon(QIcon("img:play_circle.svg"))
+        self.run_button.setToolTip(EXPERIMENT_IS_MANUAL_UPDATE_MESSAGE)
         self.run_button.setIconSize(QSize(32, 32))
         self.run_button.clicked.connect(self.run_experiment)
-        self.run_button.setToolButtonStyle(Qt.ToolButtonStyle.ToolButtonTextBesideIcon)
+        self.run_button.setToolButtonStyle(Qt.ToolButtonStyle.ToolButtonIconOnly)
```

Comment:
Only a suggestion, but I felt the play button seemed a bit harsh;

```suggestion
        self.run_button.setToolButtonStyle(Qt.ToolButtonStyle.ToolButtonIconOnly)
        self.run_button.setMinimumWidth(60)
        self.run_button.setMinimumHeight(40)
        self.run_button.setStyleSheet("""
            border-radius: 8px; /* Adjust the radius to get the desired roundness */
            background-color: qlineargradient(
                x1:0, y1:0, x2:0, y2:1,
                stop:0 #f0f0f0, /* Light grey color at the top */
                stop:1 #d9d9d9  /* Slightly darker grey color at the bottom */
            );
            border: 1px solid #bfbfbf; /* Optional: border color */
            padding: 5px; /* Optional: inner padding */
        """)
```

![Screenshot 2024-09-13 at 12 18 38](https://github.com/user-attachments/assets/a1ba7520-4897-47cf-af4c-18980c7a224e)

We could (and perhaps should) increase the size of the QComboBox to make them more similar in size.
That would also resolve some of the visual discrepancy between the size of the QComboBox and the visual style of the menu presented when clicked.

Reply:
Thanks for the tips btw Andreas :)